test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -45,3 +45,27 @@
                      "service-status-version" 2
                      "status" "foo status 2"}}
              (get body "services"))))))
+
+(deftest single-service-status-endpoint-test
+  (with-app-with-config
+    app
+    [jetty9-service/jetty9-service
+     webrouting-service/webrouting-service
+     status-service
+     foo-service]
+    {:webserver {:port 8180
+                 :host "0.0.0.0"}
+     :web-router-service {:puppetlabs.trapperkeeper.services.status.status-service/status-service "/status"}}
+    (testing "returns service information for service that has registered a callback"
+      (let [req (http-client/get "http://localhost:8180/status/v1/services/foo")]
+        (is (= 200 (:status req)))
+        (is (= {"service-version" "1.1.0"
+                "service-status-version" 2
+                "status" "foo status 2"}
```

Comment:
I'd lean towards including it. I think the user can definitely get by without it since they'll need it anyway to query a specific service's status, but I think it might be a little more convenient to have it in the body.